docs/meshstack.monitoring.md

```diff
@@ -1,16 +1,67 @@
 ---
 id: meshstack.monitoring
-title: Metrics & Monitoring
+title: Monitoring & Telemetry
 ---
 
-In addition to [Logging & Auditing](./meshstack.logging.md), meshStack supports metric monitoring of production
-deployments using [Prometheus](https://prometheus.io/). Our [managed service](meshstack.managed-service.md) team
-relies on this monitoring to operate meshStack installations.
+Our [managed service](meshstack.managed-service.md) team
+relies on monitoring & telemetry to operate meshStack installations for our customers. The integration of these systems are a mandatory pre-requisite for using our managed services.
 
-## Prometheus Scraping
+This page documents what data we collect and for what purposes we process it. All customer data we collect and store is handled with confidentiality in accordance with our information security management system as described in our [FAQ](./faq.md).
 
-Prometheus scrapes metrics from nodes (for IaaS deployments) as well as from meshStack components.
-Scraping is secured using HTTPS Basic auth. The credentials are managed as part of the deployment. Information about metrics can be found in [Metrics Metadata](./meshstack.monitoring.metrics.md).
+## Customer data
+
+Customer data is any data entered into meshStack via meshPanel or the API and also includes any data that meshStack retrieves from the customer's cloud platforms like cloud tenant metadata or usage reports.
+
+## Metrics
+
+Metrics are labelled, numerical time-series data.
+
+- operating-system level metrics like CPU and disk usage
+- application-service metrics like database, queue and load-balancer performance
+- meshStack application metrics like API performance and statistics about meshObjects like meshCustomers, meshProjects etc.
+
+Application-level metrics can contain labels that include customer data such as meshCustomer and meshPlatform identifiers.
+
+## Logs
+
+meshStack collects various logs as described in [Logging & Auditing](./meshstack.logging.md). Logs can contain customer data and also PII like user identifiers.
+
+Log data is therefore not part of telemetry data we collect.
+
+## Product Usage Data
+
+meshPanel can optionally collect product usage data and feedback. We use this data to help us understand how to improve the product and to prioritize new features.
+meshPanel uses the privacy-preserving and open source web-analytics library plausible.io for collecting anonymous usage data as described in the [plausible data policy](https://plausible.io/data-policy).
```

Comment:
maybe mention it is self-hosted to add even more security assurance.

Reply:
agree in the email I put "Data will remain within the secure meshcloud environment and will not be shared or processed externally. "